src/main/java/io/antmedia/datastore/db/types/Subscriber.java

```diff
@@ -82,6 +81,18 @@ public class Subscriber {
 	@ApiModelProperty(value = " count of subscriber usage")
 	private int concurrentConnectionsLimit = 1;
 
+	private boolean playBlocked = false;
+
+	private boolean publishBlocked = false;
+
+	private long playBlockTime = 0;
+
+	private long publishBlockTime = 0;
+
+	private long playBlockedUntilTime = 0;
+
+	private long publishBlockedUntilTime = 0;
```

Comment:
Please add a field that shows the instance where the subscriber is watching the stream

Reply:
done